public/index.html

```diff
@@ -14,7 +14,7 @@
   <meta property="og:type" content="website">
   <meta property="og:image" content="/img/cover.png">
   <meta name="theme-color" content="#29363b" />
-  <meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src 'self'; connect-src 'self' https://send.fs.neo.org; style-src 'unsafe-inline' 'self' https://fonts.googleapis.com; font-src https://fonts.gstatic.com; img-src 'self'; form-action 'self';  base-uri 'self';">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src 'self'; connect-src 'self' https://rest.t5.fs.neo.org; style-src 'unsafe-inline' 'self' https://fonts.googleapis.com; font-src https://fonts.gstatic.com; img-src 'self'; form-action 'self';  base-uri 'self';">
```

Comment:
Hmm. It was never like that, we always proxy everything through send.fs.neo.org.